back/spec/mailers/user_blocked_mailer_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe UserBlockedMailer do
+  include_context 'when user_blocking duration is 90 days' do
```

Comment:
This context is included just to get tests green on CI, but will not removed by changes in #4252